website/search/elastic_search.py

```diff
@@ -707,13 +707,14 @@ def update_user(user, index=None):
         'normalized_user': normalized_names['fullname'],
         'normalized_names': normalized_names,
         'names': names,
-        'job': user.jobs[0]['institution'] if user.jobs else '',
-        'job_title': user.jobs[0]['title'] if user.jobs else '',
-        'all_jobs': [job['institution'] for job in user.jobs[1:]],
-        'school': user.schools[0]['institution'] if user.schools else '',
-        'all_schools': [school['institution'] for school in user.schools],
+        # TODO: elastic search will need to be updated to utilize the new education/employment fields
```

Comment:
Oh, I see. Is this a hard addition, grabbing the first employment and education records and filling in? Otherwise, is there a ticket for this / is this being handled in another existing ticket?